backend/app/services/splunk.py

```diff
@@ -45,6 +45,8 @@ async def query(
         """
         query["count"] = size
         query["offset"] = offset
+        query["sort_dir"] = "asc"
+        query["sort_key"] = "test_type"
```

Comment:
This isn't controlled under the `sort` query parameter? Yeah, I know the Splunk schema and query model is very different, but doesn't this mean Telco can't "really" be sorted???

Reply:
With `sort_dir` and `sort_key` Telco can be sorted but few columns are run-time generated so dropping sorting for Telco Table

Reply:
That's unfortunate -- but I suppose if it becomes important we can work that out later.

Reply:
Yeah, we can raise an issue to track it